src/content/docs/en/contributing/hacking.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Code Contributions
+description: "A step by step guide to hacking on Ultramarine Linux components"
+---
+import Alert from "../../../../components/Docs/Alert.astro";
+
+One of the most noticeable contributions you can make is code contributions. Wether it's building a new feature or fixing a bug, contributing code is a great way to make a noticeable difference to the operating system itself.
+
+This page serves as a step-by-step guide to hacking on Ultramarine Linux components.
+
+## Think of a Problem
+
+This can be one you or someone you know faces with Ultramarine, or even just computing at large.
```

Comment:
Something I would put here is, it's better to start with a problem that you yourself have or that is a known issue. New community members should probably start out with small problems first.